R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -246,8 +246,8 @@ test_that("read/write csv as DataFrame", {
   mockLinesCsv <- c("year,make,model,comment,blank",
                    "\"2012\",\"Tesla\",\"S\",\"No comment\",",
                    "1997,Ford,E350,\"Go get one now they are going fast\",",
-                   "2015,Chevy,Volt",
-                   "NA,Dummy,Placeholder")
+                   "2015,Chevy,Volt,,",
```

Comment:
I probably think that dropping the extra tokens in the **longer** case  is an incorrect behaviour by referring the json behaviour. But, I know this change could affect current users, so we might need to do something for that, e.g., adding a new option to keep the current behaviour. WDYT? cc: @HyukjinKwon